tools/benchmark/cmd/watch.go

```diff
@@ -137,8 +137,8 @@ func watchFunc(cmd *cobra.Command, args []string) {
 	<-pdoneC
 
 	// put phase
-	// total number of puts * number of watchers on each key
-	eventsTotal = watchPutTotal * (watchTotal / watchedKeyTotal)
+	// total number of puts * (number of watchers on each key) / (number of events triggered per put)
+	eventsTotal = watchPutTotal * (watchTotal / watchedKeyTotal) / (watchedKeyTotal / watchKeySpaceSize)
```

Comment:
`numWatchers` would be a `map[string]int` that counts the number of watchers for a given key